website/content/docs/troubleshooting.mdx

```diff
@@ -58,3 +58,110 @@ $ docker volume list
 ```
 
 Locate the volumes named starting with `pack-cache-` and remove them with `docker volume rm`.
+
+# Investigating deployed applications
+
+Waypoint includes several commands to support debugging and monitoring while
+developing your application.
+
+## Exec into the application container
+
+After deploying your application, you can use `waypoint exec` to run
+commands in the context of the most recent deployment. Typically, `waypoint
+exec` will be used for running database migrations and debugging. However, you
+can use it for any purpose.
+
+Use the `exec` command to open a shell prompt.
+
+```shell-session
+$ waypoint exec /bin/bash
+```
+
+Since you are in the deployment directory, you will observe that Waypoint
+automatically executes against the currently deployed application.
+
+From within the Docker container, validate that this is the actual application
+by listing out the directory hosting the application's compiled files.
```

Comment:
I'm not clear on what we mean by `deployment directory` here. I associate `the currently deployed application` with the `/workspace` dir which contains my application , but `exec /bin/bash` drops me into `/`. Based on the suggestion below to `cd /`, I may be experiencing slightly different behavior than you?

Reply:
You're right! Maybe this behavior has changed since the Learn guide was written? I'm also dropped into `/` when I run a `waypoint exec /bin/bash`. :thinking: Maybe it's there just to ensure readers are always in the right directory so that `ls` works? I'm happy to just change it to `ls` though.

Reply:
```suggestion
by listing out the directory hosting the application's compiled files. For example, if you used the `pack` builder this directory will be under `/workspace`.
```